### PR TITLE
samples, raise test coverage back to 94%, fixes to transformer

### PR DESCRIPTION
**Pull request by eclendenning-va**
### This is still not the complete transformer. ### 
@ilaflamme-va  is completing the extensions in the branch API-8652-organization-transformer-continued.

This adds in samples for test coverage, and provides minor fixes to the transformer.

### CODE REVIEW

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java [hunk 1]
```diff
@@ -176,8 +178,8 @@ private Address address(
         entry.internal(InsuranceCompany.CLAIMS_INPT_PROCESS_STATE).orElse(null),
         entry.internal(InsuranceCompany.CLAIMS_INPT_PROCESS_ZIP).orElse(null),
         "RXCLAIMS",
-        entry.internal(InsuranceCompany.CLAIMS_RX_PHONE_NUMBER).orElse(null),
-        entry.internal(InsuranceCompany.CLAIMS_RX_FAX).orElse(null),
+        entry.internal(InsuranceCompany.CLAIMS_INPT_PHONE_NUMBER).orElse(null),
```
Comment (bschofield-va) [inline vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java:181]: Any fields read, like these, must be in that big ol' `REQUIRED_FIELDS` list or they won't be requested for the RPC.

The `REQUIRED_FIELDS` list is how the transformer communicates with the controller to tell it what data is needed.

Comment (bschofield-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java:181]: @ilaflamme-va @eclendenning-va @jhulbert-va @jabando-va

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java [hunk 2]
```diff
@@ -324,7 +325,7 @@ private Address address(
 
   private List<ContactPoint> organizationTelecom(String phoneNumber) {
     if (isBlank(phoneNumber)) {
-      return Collections.emptyList();
+      return emptyList();
```
Comment (jhulbert-va) [inline vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java:328]: Not sure this is right. Even if we don't have a telecom, the fhir object is going to return an empty list. (Se the line in `toOrganization`.

Comment (bschofield-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationTransformer.java:328]: Let's address this in Transformer 3: Dark of the Moon, I'd like to go ahead and combine these changes with the controller to move forward with it.

**Review by bschofield-va — APPROVED** [review]
Keep reviewing and make the changes in part 3.

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**